image/manifest.go

```diff
@@ -89,7 +89,7 @@ func (m *manifest) validate(w walker) error {
 
 func (m *manifest) unpack(w walker, dest string) error {
 	for _, d := range m.Layers {
-		if d.MediaType != string(schema.MediaTypeImageConfig) {
+		if d.MediaType != string(schema.MediaTypeImageLayer) {
 			continue
```

Comment:
@wking  if the layers MUST have a particular type is not said in the spec currently, I think we should make it clear in the spec first.